src/delegates/extension/lazyExtension/Rationale.md

```diff
@@ -0,0 +1,6 @@
+Correct answer: **will not compile**
```

Comment:
The explanation doesn't really point out why. Is it because the signature of `lazy` takes a `()->T`, which makes `MyClass` instance not accessible, which is kind of implied in the last point?

Reply:
actually, having the instance available in the lambda wouldn't really help - that's actually the point of my second example.
I'll try to rephrase things a bit to make the issue clearer